packages/rum-recorder/package.json

```diff
@@ -7,7 +7,7 @@
   "types": "cjs/index.d.ts",
   "scripts": {
     "build": "run-p build:cjs build:esm build:bundle",
-    "build:bundle": "rm -rf bundle && webpack --config webpack.config.js --mode=production && yarn replace-build-env bundle",
+    "build:bundle": "if [[ $WITH_SUFFIX != true ]]; then rm -rf bundle && webpack --config webpack.config.js --mode=production && yarn replace-build-env bundle; fi",
```

Comment:
what about moving `build:bundle` scripts to a cli command?
it could help to document this behavior